src/utils/polling.js

```diff
@@ -61,20 +97,31 @@ async function pollSubscriptionTopics() {
       auth: { bearer: accessToken },
     };
 
+    const resourceToPoll = namedEventToResourceType(topic);
+    const mostRecentPoll = db.select('polling', (p) => p.resource === resourceToPoll);
+    const lastUpdated = mostRecentPoll.length === 0 ? null : mostRecentPoll[0].timestamp;
     const fhirClient = mkFhir(options);
     fhirClient
-      .search(getSearchQuery(topic))
+      .search(getSearchQuery(resourceToPoll, lastUpdated))
       .then((response) => {
         const { data } = response;
 
+        // Add poll to DB
+        const poll = {
+          id: uuidv4(),
+          timestamp: new Date(Date.now()).toISOString(),
+          resource: resourceToPoll
+        };
+        db.insert('polling', poll);
```

Comment:
Are you using inset here instead of upsert to keep a log? I think we should be careful since line 101 is just a pure select... we shouldn't rely on how loki orders entries by default. Especially since the db abstraction means the actual database should be easily replaceable this could potentially break. IMO this should use upsert (and then use another logging collection to maintain an audit trail)

Reply:
Good catch, on line 101 I was going to sort by the timestamp but forgot to. Do you think we need to keep a log? If not I can just change this to `upsert`.

Reply:
There's already a log in the console (tbh no idea where its stored but I'm assuming it is) so I think using upsert here is fine